cypress/support/fragments/exportManager/exportManagerSearchPane.js

```diff
@@ -0,0 +1,113 @@
+import { Pane, Button, TextField, MultiColumnListCell, Accordion, Checkbox, Modal } from "../../../../interactors";
+import { including } from "bigtest";
+
+const searchButton = Button({ type: 'submit' });
+const userSearchResults = Pane('User Search Results')
+const startTimeAccordion = Accordion({ id: 'startTime' });
+const endTimeAccordion = Accordion({ id: 'endTime' });
+const systemAccordion = Accordion({ id: 'isSystemSource' });
+const sourceAccordion = Accordion({ id: 'createdByUserId' });
+const jobTypeAccordion = Accordion({ id: 'type' });
+const statusAccordion = Accordion({ id: 'status' });
+const getSearchResult = (row = 0, col = 0) => MultiColumnListCell({ 'row': row, 'columnIndex': col });
+
+export default {
+	getSearchResult,
+	waitLoading() {
+		cy.expect(Pane('Export jobs').exists());
+	},
+	searchById(id) {
+		cy.do([
+			TextField().fillIn(id),
+			searchButton.click(),
+		])
+	},
+	selectSearchResultItem(indexRow = 0) {
+		return cy.do(this.getSearchResult(indexRow, 0).click());
+	},
+	closeExportJobPane() {
+		cy.do(Button({ ariaLabel: 'Close Export job ' }).click());
+	},
+	resetAll() {
+		cy.do(Button('Reset all').click());
+		// Cypress clicks before the UI loads, there is no way to attach waiter to element
+		cy.wait(1000);
```

Comment:
I think it will be better if we create new function like waitClick() in const and put this waiter in it. We can use this function in another function

Reply:
done